### PR TITLE
DOCSP-45210 What's New

### PR DESCRIPTION
**Pull request by stephmarie17**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-45210>

### Staging Links

<li><a href=https://deploy-preview-99--docs-ruby.netlify.app/whats-new>whats-new</a></li><li><a href=https://deploy-preview-99--docs-ruby.netlify.app/index>index</a></li>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  support for JRuby 9.4.
+- Supports the newly-released Ruby-BSON version 5.0.
+- Allows connection strings without a slash between the hosts and the options.
+  For example, "mongodb://example.com?w=1" and "mongodb://example.com/?w=1" are
+  both valid connection strings now.
+- Sends container runtime and orchestration metadata for the client environment
+  to the server for analytics purposes.
+- Writes a warning message to the log when detecting the host as a CosmosDB
+  (Azure) or DocumentDB (Amazon) instance.
+- Attempts retries of read or write operations on a different mongos instance in
```
Comment (lindseymoore) [inline source/whats-new.txt:59]:
```suggestion
- Attempts retries of read or write operations on a different ``mongos`` instance in
```
Suggestion: I normally see ``mongos`` in monospace.
https://www.mongodb.com/docs/manual/core/sharded-cluster-query-router/

File: source/whats-new.txt [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+------------------
+
+The {+driver-short+} 2.20 release includes the following new features:
+
+- Discontinues support for Ruby 2.5 and 2.6. Deprecates support for Ruby 2.7 and
+  JRuby 9.2, which will be discontinued in the next minor driver version. Adds
+  support for JRuby 9.4.
+- Supports the newly-released Ruby-BSON version 5.0.
+- Allows connection strings without a slash between the hosts and the options.
+  For example, "mongodb://example.com?w=1" and "mongodb://example.com/?w=1" are
```
Comment (lindseymoore) [inline source/whats-new.txt:53]:
```suggestion
  For example, ``mongodb://example.com?w=1`` and ``mongodb://example.com/?w=1`` are
```
Suggestion: I normally put connection strings in monospace. I don't see this in the style guide, but it's also a bit easier to read when it's distinguished from the rest of the text.

**Review by lindseymoore — APPROVED** [review]
LGTM! 2 small suggestions

File: source/whats-new.txt (hunk 2 above)
Comment (stephmarie17) [reply source/whats-new.txt:53]: I was going to do this as well! I copied this content over from the old release notes page so left as-is, but I agree with you and will update.

File: source/whats-new.txt (hunk 1 above)
Comment (stephmarie17) [reply source/whats-new.txt:59]: Agree, updating!

File: source/whats-new.txt [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+The {+driver-short+} 2.21 release includes the following new features: 
+
+- Supports the Client-Side Operations Timeout (CSOT) feature, which unifies
+  most timeout-related options under a single ``timeout_ms`` option.
+- Supports {+mdb-server+} version 8.0.
+- Support for range v2 queries with Queryable Encryption, including a new ``trim_factor``
+  parameter. For more information about Queryable Encryption, see :manual:`Queryable Encryption </core/queryable-encryption>`
+  in the {+mdb-server+} manual.
+
```
Comment (stephmarie17) [inline source/whats-new.txt:40]: [question for technical reviewer] Does your team also want to link to driver release notes for each version at the end of the summary? Not all "What's New" pages do this but I noticed to the [current docs page](https://www.mongodb.com/docs/ruby-driver/current/release-notes/) links out so wanted to verify the preference of the team.

Comment (stephmarie17) [reply source/whats-new.txt:40]: See the [Node What's New docs](https://www.mongodb.com/docs/drivers/node/current/whats-new/#what-s-new-in-6.12) as an example of where the link would go.

Comment (jamis) [reply source/whats-new.txt:40]: Linking to the specific versions might be most convenient for the user, if that's not too much trouble for the docs team.

@alexbevi might have some thoughts?

Comment (stephmarie17) [reply source/whats-new.txt:40]: It's no trouble 👍

Comment (stephmarie17) [reply source/whats-new.txt:40]: Updated the PR with the links in case @alexbevi wants to take a look and lmk if that looks good to go.

Comment (alexbevi) [reply source/whats-new.txt:40]: I like the links to the GH release. LGTM

**Pull request merged**